### DISCUSSION TITLE
Cobbler reassembly

### DISCUSSION THREAD

**Discussion by 1392273211** [cobbler/cobbler#3608] [Q&A]
Can you provide other reinstallation methods besides koan when reinstalling the system on cobbler

**Comment by SchoolGuy** [comment]
The default case for installing systems with Cobbler is to use PXE or HTTPboot to install them via the network.

A last alternative is to use `cobbler buildiso` to create a bootable image that can be copied onto a USB Stick. Koan is only used for systems that are neither available physically to use an ISO nor have network boot available to them.

**Reply by 1392273211** [reply]
Cobbler buildiso did not find the corresponding configuration on the Internet. Is there any corresponding documentation in the community?

**Reply by SchoolGuy** [reply]
We have two pieces of information:

- https://cobbler.readthedocs.io/en/v3.3.3/user-guide/building-isos.html#building-isos
- https://cobbler.readthedocs.io/en/v3.3.3/cobbler.html?highlight=buildiso#cobbler-buildiso
